CourseGrab/ViewControllers/Home/HomeViewController.swift

```diff
@@ -0,0 +1,487 @@
+//
+//  HomeViewController.swift
```

Comment:
Why does this file exist? It seem like a copy of `HomeTableViewController`.

Reply:
It was something I was doing for with SkeletonView but scrapped it. Seems like I removed the reference but not the file itself.